jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -227,6 +227,7 @@ with what features/services are supported.
 
         <registryKey key="aws.credentialProcess.timeout" description="AWS Credential Process timeout (ms)" restartRequired="false" defaultValue="30000"/>
         <registryKey key="aws.debuggerAttach.timeout" description="Time allowed for debuggers to attach before timing out (ms)" restartRequired="false" defaultValue="60000"/>
+        <registryKey key="aws.feature.cloudDebug" description="Enables the Cloud Debug for ECS set of features" restartRequired="false" defaultValue="true"/>
```

Comment:
can we call it `aws.feature.ecsCloudDebug`

Reply:
Yep!